web_external/templates/widgets/studyResultsStudyDetailPage.jade

```diff
@@ -4,6 +4,10 @@
     - var featureset = model.get('featureset')
     table.table.table-condensed.isic-study-results-study-details-table
       tr
+        td Unique ID
```

Comment:
I realized this should have the `.isic-study-results-study-details-table-label` class.